docs/link-manager.md

```diff
@@ -0,0 +1,43 @@
+## Link Manager
+
+Use the Link Manager instead of an internal router to: 
+
+- route inside micro front-ends 
+- reflect the route 
+- keep the navigation state in Luigi
+
+Here is a code sample and a definition of the different parameters:
+
+````
+// navigation
+LuigiClient.linkManager().navigate('/overview')
+LuigiClient.linkManager().navigate('users/groups/stakeholders')
+LuigiClient.linkManager().fromClosestContext().navigate('/users/groups/stakeholders')
+LuigiClient.linkManager().fromContext('project').navigate('/settings')
+ 
+ 
+// additionally send parameters to route
+LuigiClient.linkManager().fromClosestContext().withParams({foo: 'bar'}).navigate('settings')
+ 
+ 
+// feature: preserve view to go back afterwards
+LuigiClient.linkManager().fromContext('project').navigate('/settings', null, true) // preserve view
+LuigiClient.linkManager().hasBack()
+LuigiClient.linkManager().goBack({foo: 'bar})
+````
+
+## Navigation
+
+- **navigate(path)** contains either a full absolute path or a relative path without a leading slash that uses the active route as a base. This is a classical navigation.
+- **fromClosestContext()** navigates from the closest parent node that has the ****navigationContext** declared in its navigation configuration. The node is usually one with a dynamic path segment.
```

Comment:
1. `Node`
2. ~The node is usually one with a dynamic path segment.~ Referring to your comment:
`The **navigationContext** is used only in combination with a dynamic path segment.`